sdk/storage/storage-file/review/storage-file.api.ts

```diff
@@ -0,0 +1,787 @@
+  constructor(url: string, credential?: Credential, options?: NewPipelineOptions);
+  createShare(shareName: string, options?: ShareCreateOptions): Promise<{
+          shareCreateResponse: Models.ShareCreateResponse;
+          shareClient: ShareClient;
+      }>;
+  deleteShare(shareName: string, options?: ShareDeleteMethodOptions): Promise<Models.ShareDeleteResponse>;
+  static fromConnectionString(connectionString: string, options?: NewPipelineOptions): FileServiceClient;
+  getProperties(options?: ServiceGetPropertiesOptions): Promise<Models.ServiceGetPropertiesResponse>;
+  getShareClient(shareName: string): ShareClient;
+  listShares(options?: ServiceListSharesOptions): PagedAsyncIterableIterator<Models.ShareItem, Models.ServiceListSharesSegmentResponse>;
+  setProperties(properties: Models.FileServiceProperties, options?: ServiceSetPropertiesOptions): Promise<Models.ServiceSetPropertiesResponse>;
+}
+
+// @public
+interface FileSetHTTPHeadersOptions extends FileAndDirectorySetPropertiesCommonOptions, CommonOptions {
+  abortSignal?: AbortSignalLike;
+}
+
+// @public
+interface FileSetMetadataOptions extends CommonOptions {
+  abortSignal?: AbortSignalLike;
+}
+
+// @public
+interface FileStartCopyOptions extends CommonOptions {
+  abortSignal?: AbortSignalLike;
+  metadata?: Metadata;
+}
+
+// @public
+class FileSystemAttributes {
+  archive: boolean;
+  directory: boolean;
+  hidden: boolean;
+  none: boolean;
+  noScrubData: boolean;
+  notContentIndexed: boolean;
+  offline: boolean;
+  static parse(fileAttributes: string): FileSystemAttributes;
+  readonly: boolean;
+  system: boolean;
+  temporary: boolean;
+  toString(): string;
+}
+
+// @public
+interface FileUploadRangeFromURLOptions extends Models.FileUploadRangeFromURLOptionalParams, CommonOptions {
+  abortSignal?: AbortSignalLike;
+}
+
+// @public
+interface FileUploadRangeOptions extends CommonOptions {
+  abortSignal?: AbortSignalLike;
+  contentMD5?: Uint8Array;
+  progress?: (progress: TransferProgressEvent) => void;
+}
+
+// @public
+export function generateAccountSASQueryParameters(accountSASSignatureValues: AccountSASSignatureValues, sharedKeyCredential: SharedKeyCredential): SASQueryParameters;
+
+// @public
+export function generateFileSASQueryParameters(fileSASSignatureValues: FileSASSignatureValues, sharedKeyCredential: SharedKeyCredential): SASQueryParameters;
+
+// @public
+class HttpHeaders {
+  constructor(rawHeaders?: RawHttpHeaders);
+  clone(): HttpHeaders;
+  contains(headerName: string): boolean;
+  get(headerName: string): string | undefined;
+  headerNames(): string[];
+  headersArray(): HttpHeader[];
+  headerValues(): string[];
+  rawHeaders(): RawHttpHeaders;
+  remove(headerName: string): boolean;
+  set(headerName: string, headerValue: string | number): void;
+  toJson(): RawHttpHeaders;
+  toString(): string;
+}
+
+// @public
+interface HttpOperationResponse extends HttpResponse {
+  blobBody?: Promise<Blob>;
```

Comment:
Consider renaming based on outcome of blobs discussion.

Reply:
`bodyAsBlob` seems reasonable fwiw.

Reply:
`contentAsBlob`? Closing on the consistency meeting.

Reply:
#6020